jerry-ext/extfunc/extfunc-print.c

```diff
@@ -0,0 +1,112 @@
+/* Copyright JS Foundation and other contributors, http://js.foundation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "jerryscript-ext/extfunc.h"
```

Comment:
I wonder why scattering those functions into separate subdirectories and headers (sort of: separate sub-namespaces)? I.e., "args" compiles everything related to argument handling, validation, conversion, etc. Should common but non-standard functions be not kept together but each split to a different namespace? (If it's only about not pulling in unnecessary implementations into the app binary, the linker will do that automatically if the implementations are in separate sources, even if the declarations are in the same header. Declared but unreferenced functions will not cause the linking of the defining objects.)